features/ei-analytics-feature/org.wso2.analytics.solutions.ei.analytics.feature/src/main/resources/dashboards/ei-anlaytics.json

```diff
@@ -2784,7 +2771,7 @@
                                         "defaultValue": "admin"
                                     }
                                 ],
-                                "query": "from ESBStatAgg on not(AGG_TIMESTAMP is null) within 1530816178000L, 1530816609000L per \u0027seconds\u0027 select sum(noOfInvocation) as Count, sum(faultCount) as Faults, sum(noOfInvocation - faultCount) as Success, AGG_TIMESTAMP as Time, AGG_TIMESTAMP as date group by AGG_TIMESTAMP"
+                                "query": "from MediatorStatAgg on true AND componentName\u003d\u003d\u0027test\u0027 within 331237800000L, 394309800000L per \u0027minutes\u0027 select sum(noOfInvocation) as Count, sum(faultCount) as Faults, sum(noOfInvocation - faultCount) as Success, AGG_TIMESTAMP as Time group by AGG_TIMESTAMP"
```

Comment:
Why there is a on condition for true? 
from MediatorStatAgg on true AND componentName=test

Reply:
This is because of the default values. 'true' value represent a hash object attribute, entryPoint. If entryPoint is not available, it should be neglected. In the default scenario its not available.

Reply:
Ack